x-pack/plugin/sql/build.gradle

```diff
@@ -1,5 +1,13 @@
-evaluationDependsOn(xpackModule('core'))
-
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+/*
```

Comment:
Looks like we snuck in a second header.

Reply:
double the copyright !  good spot.

Reply:
decided to remove all headers... so this is now gone.